app.R

```diff
@@ -99,7 +99,7 @@ ui <- dashboardPage(
                   selectInput(
                     inputId = "template_type",
                     label = "Template:",
-                    choices = list("ScRNA-seqAssay", "Demographics", "Diagnosis", "FamilyHistory", "Exposure", "FollowUp", "Therapy")
+                    choices = list("ScRNA-seqAssay", "BulkRNA-seqAssay", "BulkRNA-seqAlignment", "Demographics", "Diagnosis", "FamilyHistory", "Exposure", "FollowUp", "Therapy")
```

Comment:
Let's not include the bulk RNA-seq assay for now; it can just be commented out until I fix the bulk RNA schema; there are a couple of attributes that need to be updated o.w. people might freak out.